BIDVerifiableCredential.js

```diff
@@ -100,6 +100,58 @@ const requestVCForID = async (tenantInfo, type, document) => {
     }
 }
 
+const requestVCForPayload = async (tenantInfo, type, payload) => {
```

Comment:
interface should be 
`(tenantInfo, type, userDid, userPublicKey, issuer, info)`